tests/tensor_search/integ_tests/test_search_combined.py

```diff
@@ -644,3 +642,82 @@ def test_search_vectoriseIsCalledWithEnableCacheTrue(self):
                     args, kwargs = mock_vectorise.call_args
                     self.assertTrue(kwargs["enable_cache"])
                 mock_vectorise.reset_mock()
+
+    def test_empty_lexical_query(self):
+        for index in [self.structured_default_text_index, self.unstructured_default_text_index]:
+            with self.subTest(index=index.type):
+                tensor_search.add_documents(
+                    config=self.config,
+                    add_docs_params=AddDocsParams(
+                        index_name=index.name,
+                        docs=[
+                            {"_id": "1", "text": "document_1"},
```

Comment:
Try to make your test as simple and concise as possible. Having so many documents here doesn't add any value. I think 3-4 docs would suffice.

Reply:
Revised to four documents!